.travis.yml

```diff
@@ -13,21 +13,8 @@ addons:
 install:
   - pip install tox
 
-env:
```

Comment:
the reason we had the former was to get separate (and parallel) builds in travis